Modules/vc-module-pricing/VirtoCommerce.PricingModule.Web/Scripts/blades/pricelist-item-list.js

```diff
@@ -154,6 +154,38 @@
                 return $scope.gridApi && _.any($scope.gridApi.selection.getSelectedRows());
             },
             permission: blade.updatePermission
+        },
+        {
+            name: "platform.commands.export",
+            icon: 'fa fa-upload',
+            canExecuteMethod: function () {
+                return true;
+            },
+            executeMethod: function () {
+                var selectedRows = $scope.gridApi.selection.getSelectedRows();
+                if (selectedRows && selectedRows.length > 0) {
+                    var productIds = _.map(selectedRows, function (product) {
+                        return product.productId;
+                    });
+                }
+
+                var dataQuery = {
```

Comment:
Can we have one method for building search criteria (e.g. on [refresh](https://github.com/VirtoCommerce/vc-platform-core/pull/377/files#diff-b118ca6a7478d87ad2d8ad8167b8b5c5R9) - need to create a method here) and for building dataQuery with something like
```
var dataQuery = {
 exportTypeName: "Price",
};
 dataQuery = angular.extend(dataQuery, GetSearchCriteria());
```

Reply:
You could get search criteria from these lines
https://github.com/VirtoCommerce/vc-platform-core/pull/377/files/b74a7572f39aa7468a732b3ca622dcf6996ec7ce..abaa851c15ac15e933ba8008cd87f320d280bb99#diff-b118ca6a7478d87ad2d8ad8167b8b5c5L9
and use it in both places.